paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziCallback.kt

```diff
@@ -74,7 +74,7 @@ internal class PaparazziCallback(
             } else if (type.isArray && type.componentType == Int::class.javaPrimitiveType) {
               // Ignore.
             } else {
-              logger.error(null, "Unknown field type in R class: $type")
+              logger.error(null, "Unknown field type ($type) in R class: $field")
```

Comment:
```suggestion
              logger.error(null, "Unknown field type ($type) for field (${field.name}) in class: ${field.declaringClass.name}")

```

Reply:
Original: `Unknown field type in R class: int`
PR: `Unknown type (int) in R class: public static int app.cash.paparazzi.sample.R$color.cameraBody`
Suggestion:  `Unknown type (int) for field (cameraBody) in R class: app.cash.paparazzi.sample.R$color`

Reply:
I would rather not go more specific, the original had the same problem, it hides info. `private` and `transient` might give more clues, as seen in the original issue. And we don't know what else we would be hiding by not using toString directly from Java. Since this is not a normal behaviour, but an edge case, the more info we get, the better, and the format doesn't really matter.

I would be happy to settle on a compromise, which reads better, but doesn't hide:
```suggestion
logger.error(null, "Unknown field type $type for field ${field.name} in class: ${field.declaringClass.name}, full signature: ${field}.")
```